### PR TITLE
EN: Update some apc_chat

### PR DESCRIPTION
**Pull request by HeroNCHAT**

### CODE REVIEW

File: Dialogue/np_012540.csv [hunk 1]
```diff
@@ -12,13 +12,13 @@ favs_present_080#0,"""Received \""<%skit_text>\"" from Aika."""
...
-skit01_030#0,"""機会があれば、積極的に<br>取得を目指してほしい。<br>そのための仕組みだからな。"""
-11_voice_NPC_5254_100#1,"""地球は、私にとっても大切な場所。<br>大事な思い出を残してきた場所だ。"""
-st01_020#0,"""言ってしまえば、地球は私にとって<br>もう一つの故郷ともいえる。<br>守りたいと思うのは、当然だ。"""
-11_voice_NPC_5254_104#0,"""私が言うのもなんだが<br>司令の情報収集力には驚かされる。"""
-st01_020#1,"""こちらが隠しておきたいことも<br>司令は知っていたりする。<br>とても……その、厄介だ。"""
+skit01_010#0,"""It looks like you can get it. Here is it, A special gift."""
+skit01_020#0,"""If you can get a bonus key from that special gift you can challenge a bonus quest."""
+skit01_030#0,"""If the opportunity arises, Take the initiative quickly get it. That's is the way of doing it."""
+11_voice_NPC_5254_100#1,"""The earth is an significant place for me. There're so many important memories that i left there."""
+  st01_020#0,"""In other words, for me, the earth can be considered as my another homeland. That is natural when I want to protect it."""
```
Comment (TheBlazter) [inline Dialogue/np_012540.csv]: There is an space before the line ID (it cause an error)

File: Dialogue/apc_chat.csv [hunk 2]
```diff
@@ -7180,28 +7180,28 @@ MemberDead#107,"""……赦しはしない"""
...
-CondPoison#108,"""毒とは……不覚です"""
-CondPanic#108,"""頭の中がふわふわして……<br>何が何だかわかりません……"""
-CondStan#108,"""身体が動きません……"""
-CondMirage#108,"""これは……まやかし？"""
-Die#109,"""道半ばで……不覚です……"""
+DestroyEnemy2#108,"""Say farewell"""
+DestroyEnemy3#107,"""See ya next time"""
+CondFreeze#108,"""Frozen...already"""
+CondBurn#108,"""Bur...Burned!"""
+CondShock#108,"""Electric shock...so uncomfortale"""
```
Comment (alama) [inline Dialogue/apc_chat.csv]: uncomfortale? or uncomfortable?

File: Dialogue/apc_chat.csv [hunk 3]
```diff
@@ -7304,22 +7304,22 @@ IncidentJsApcBegin#99,"""/ci1 1 おや、良いところに！<br>すまぬが
...
-EnemyBoost#110,"""ちっ、メンドクセェ感じの<br>ヤツが出てきてやがるな……！"""
-DestroyEnemy1#110,"""雑魚が！"""
-DestroyEnemy2#110,"""邪魔だ！"""
-DestroyEnemy3#109,"""うざってぇんだよ！"""
-CondFreeze#110,"""凍結だと……っざけんな！"""
-CondBurn#110,"""燃焼だと……っざけんな！"""
+EnemyWeak#110,"""Kuh, weak like a small fish"""
+EnemyLarge#110,"""Happy?! nothing more than the body"""
+EnemyBoss#110,"""I guess these guys finally came out"""
+EnemyRare#110,"""Whut? Awh, it's a rare person?"""
```
Comment (alama) [inline Dialogue/apc_chat.csv]: Whut? or What?
And the spell checker does not like Awh, so you may have add to _aspell/PSO2.en_Chat.dict

File: Dialogue/apc_chat.csv [hunk 4]
```diff
@@ -5299,7 +5299,7 @@ MemberDead#78,"""……許せません。<br>絶対に、許せませんわ！""
 AbilityExe#78,"""ふふふ、ここからが<br>私のターン、ですわ！"""
 AbilitySpecial#78,"""私におまかせ、ですわ！"""
 Resurrect#78,"""そろそろ起きましょうか"""
-PartyJoin#78,"""/ci1 1 私、参上ですわ！"""
+PartyJoin#78,"""/ci1 1 Here i come"""
```
Comment (alama) [inline Dialogue/apc_chat.csv]: i? or I?

File: Dialogue/np_012540.csv [hunk 5]
```diff
@@ -1,6 +1,6 @@
-favs_before_010#0,"""貴方に渡したいものがある。<br>これを受け取ってほしい。"""
-11_voice_NPC_5254_100#0,"""情報部も、司令も関係ない。<br>これは私から、個人的なお礼だ。"""
-favs_after_020#0,"""これからも、よろしく頼む。"""
+favs_before_010#0,"""I have something i want to give you. I'd like you to receive this"""
```
Comment (alama) [inline Dialogue/np_012540.csv]: i? or I?

File: add dialog [hunk 6]
```diff
@@ -0,0 +1 @@
+
```
Comment (alama) [inline add dialog]: Can you delete this file?

Comment (HeroNCHAT) [reply add dialog]: Yes, i want to delete it too but don't know how to do that.

**Pull request closed without merging**